src/Mount.cpp

```diff
@@ -1328,46 +1348,110 @@ void Mount::startSlewingToTarget()
 
 /////////////////////////////////
 //
-// stopGuiding
+// startSlewingToHome
 //
 /////////////////////////////////
-void Mount::stopGuiding()
+// Calculates movement parameters and program steppers to move to home position.
+// Takes any sync operations that have happened and tracking into account.
+void Mount::startSlewingToHome()
 {
-    stopGuiding(true, true);
+    stopGuiding();
+
+    // Make sure we're slewing at full speed on a GoTo
+    LOGV2(DEBUG_STEPPERS, F("[STEPPERS]: startSlewingToHome: Set DEC to MaxSpeed(%d)"), _maxDECSpeed);
+    _stepperDEC->setMaxSpeed(_maxDECSpeed);
+    LOGV2(DEBUG_STEPPERS, F("[STEPPERS]: startSlewingToHome: Set RA  to MaxSpeed(%d)"), _maxRASpeed);
+    _stepperRA->setMaxSpeed(_maxRASpeed);
+
+    _currentDECStepperPosition = _stepperDEC->currentPosition();
+    _currentRAStepperPosition  = _stepperRA->currentPosition();
+
+    // Take any syncs that have happened into account
+    long targetRAPosition  = -_homeOffsetRA;
+    long targetDECPosition = -_homeOffsetDEC;
```

Comment:
Technically you can remove this variable (and replace with `-_homeOffsetDEC`) but if the name is useful it should at least be `const`